src/raftstore/store/fsm/router.rs

```diff
@@ -472,166 +469,136 @@ impl<N: Fsm, C: Fsm, Ns: Clone, Cs: Clone> Clone for Router<N, C, Ns, Cs> {
 }
 
 #[cfg(test)]
-pub mod tests {
+mod tests {
     use super::*;
-    use crossbeam::channel::{SendError, TryRecvError, TrySendError};
+    use crossbeam::channel::{SendError, TrySendError};
+    use raftstore::store::fsm::batch::tests::Message;
+    use raftstore::store::fsm::batch::{self, tests::*};
```

Comment:
It's better to move the common part to test/util?

Reply:
No, they are in different crates.